Stratis.Bitcoin/Features/MemoryPool/Fee/BlockPolicyEstimator.cs

```diff
@@ -204,22 +242,28 @@ public void ProcessTransaction(TxMempoolEntry entry, bool validFeeEstimate)
             this.trackedTxs++;
 
             // Feerates are stored and reported as BTC-per-kb:
-            var feeRate = new FeeRate(entry.Fee, (int) entry.GetTxSize());
+            FeeRate feeRate = new FeeRate(entry.Fee, (int) entry.GetTxSize());
 
             this.mapMemPoolTxs.Add(hash, new TxStatsInfo());
             this.mapMemPoolTxs[hash].blockHeight = txHeight;
             this.mapMemPoolTxs[hash].bucketIndex = this.feeStats.NewTx(txHeight, feeRate.FeePerK.Satoshi);
         }
 
-        // Remove a transaction from the mempool tracking stats
-        // This function is called from CTxMemPool::removeUnchecked to ensure
-        // txs removed from the mempool for any reason are no longer
-        // tracked. Txs that were part of a block have already been removed in
-        // processBlockTx to ensure they are never double tracked, but it is
-        // of no harm to try to remove them again.
+        /// <summary>
+        /// Remove a transaction from the mempool tracking stats.
+        /// </summary>
+        /// <param name="hash">Transaction hash.</param>
+        /// <returns>Whether successfully removed transaction.</returns>
```

Comment:
please review the wording here, maybe  "Whether the transaction was successfully removed."